packages/react/src/internal/keyboard/keys.js

```diff
@@ -80,3 +80,9 @@ export const ArrowDown = {
   which: 40,
   keyCode: 40,
 };
+
+export const Delete = {
+  key: 'Delete',
+  which: 8,
+  keyCode: 8,
```

Comment:
Should we specify and handle <kbd>Delete</kbd> vs <kbd>Backspace</kbd>? I think `8` is technically Backspace and `46` is Delete? I don't have an extended keyboard with the numpad to test the Delete key directly, but I it looks like same functionality can be achieved via <kbd>fn</kbd> + <kbd>backspace</kbd> testing on https://keycode.info/.

It's super confusing to me because on a non-numpad mac keyboard they label the key as "delete" 🤦

Reply:
Hmm, that's a good point. I also used keycode.info to figure out the code for Delete. I wonder if maybe it would just be better to use a different command for keyboard controls, one that doesn't change with different type of keyboards. @andreancardona @tay1orjones I don't have an extended keyboard to test it out either.

cc:  @dakahn @vpicone  do either of y'all have an extended keyboard to test this out?